src/validate.ts

```diff
@@ -77,22 +106,27 @@ export function validate(intentRequest: object, apiResponse: object) {
     // identifies the part of the api response to validate against a schema
     const commands = apiResponse['payload']['commands'];
     const commandsLength = commands.length;
```

Comment:
Note that even checking for this before you validate against EXECUTE_RESPONSE_SCHEMA is an assumption.

Reply:
Moved these constants